manual/mactype/tools/chocolateyInstall.ps1

```diff
@@ -1,5 +1,5 @@
 $checksum = 'DF5E528205E9B293993371A4F347148E42B85C5F0472A8ED5930B9E84C4A6619'
```

Comment:
Forgot to update the checksum of the new installation file: `D4D84BBB0D67480AC9C19D9F60874A496C84C54852E12F6B85E1C8CCB8C123ED`